internal/pkg/service/buffer/storage/level/local/writer/csv/csv_test.go

```diff
@@ -21,11 +21,10 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/compression"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/volume"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/writer"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/writer/csv"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/writer/disksync"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/writer/test/testcase"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local/writer/volume"
```

Comment:
`writer` adresar obsahoval uz toho privela + potreboval som to upravit aj aby boli prehladnejsie zavislosti medzi pkgs, 
(vacsiu) cast `writer` pkg som presunul do `writer/volume` pkg.

![image](https://github.com/keboola/keboola-as-code/assets/19371734/a1c63941-fa1b-425f-b477-baf6abeeb722)